third_party/amd/lib/Dialect/TritonAMDGPU/IR/Dialect.cpp

```diff
@@ -447,6 +447,45 @@ LogicalResult ConcatOp::verify() {
   return success();
 }
 
+LogicalResult LocalLoadPackedTransposedOp::verify() {
```

Comment:
We need to add some lit tests to check the invalid cases. We can create a new AMD `invalid-attributes.mlir` like the one under `test/TritonGPU`.